sequencing/vcf_slice/dxapp.json

```diff
@@ -1,69 +1,90 @@
 {
   "name": "vcf_slice",
   "title": "Slice VCF File(s)",
-  "summary": "vcf_slice",
+  "summary": "Return a small section of a VCF file or files (similar to tabix).",
   "dxapi": "1.0.0",
-  "version": "0.0.1",
+  "version": "0.1.0",
   "categories": [
     "Variation Calling"
   ],
   "inputSpec": [
     {
       "name": "vcf_fn",
```

Comment:
"vcf_fn" -> "variants_vcfgzs",
Also need to update accordingly in the app script.